### PR TITLE
Refact/local bind action creators

### PR DESCRIPTION
**Pull request by gnandous**
There was an unnecessary usage of the whole redux [dependencie](https://github.com/edvincandon/revux/blob/master/src/utils/wrapActionCreators.js) in sources when wrapping action creators. 
Since `bindActionCreators` was the only widget used from redux, changes provided by this request intend to use a local bindActionCreators function instead. Which will thus lead projects importing `revux` to drop *redux* from their dependencies if they are not using it internally.

### CODE REVIEW

File: dist/revux.js [hunk 1]
```diff
@@ -68,9 +68,39 @@ function shallowEqual(objA, objB) {
...
+
+function bindActionCreators(actionCreators, dispatch) {
+  if (typeof actionCreators === 'function') {
+    return function () {
+      return dispatch(actionCreators.apply(undefined, arguments));
+    };
+  }
+
+  if (typeof actionCreators !== 'object' || actionCreators === null) {
+    throw new Error('bindActionCreators expected an object or a function, instead received ' + (actionCreators === null ? 'null' : typeof actionCreators) + '. ' + 'Did you write "import ActionCreators from" instead of "import * as ActionCreators from"?');
```
Comment (edvincandon) [inline dist/revux.js]: `throw new Error('[revux] - blablabla')`

Comment (gnandous) [reply dist/revux.js]: Done :D

**Comment by edvincandon** [comment]
https://github.com/reactjs/react-redux/blob/master/src/connect/mapDispatchToProps.js
react-redux also uses the `bindActionCreators` func from redux. Dropping it will lead to duplicate code (Afterall, revux is meant to be used with redux).

_Dans le contexte de magneto - je me demande s'il faut pas dropper complètement le concept de store - et connecter les components avec https://github.com/vuejs/vue-rx maybe ? On en reparle demain ?_

**Comment by gnandous** [comment]
Okay, Since revux is meant to be used along side with redux, there is not reason to bind a local action creator in deed. ;)
I'm closing the PR.

**Pull request closed without merging**